### DISCUSSION TITLE
Idea: Multi-Mod Folder Support

### DISCUSSION THREAD

**Discussion by vistabunny** [Raytwo/ARCropolis#549] [Ideas] [answered]
Posted this on GameBanana because I was thinking of a new feature that we need for ARCropolis.

[https://gamebanana.com/threads/224284](https://gamebanana.com/threads/224284)

since V4.0.0 doesn't support the Multi-mod folder but I am hoping if you guys can add Multiple Mods, and of course I do mod Super Smash Bros. Ultimate by making my own modpack, so I set up some decoy folders and the decoy files like the config.json and whatnot. because I wanted to make myself a big roster with the pack of folders in one folder.

The Feature works by Creating a folder, for your mods. name anything you want with spaces, open up the empty folder, Create a mod folder or drag and drop the mod into the empty folder. like Stage, Fighter, UI, etc.

Here's how it should look:
The Main Multi-Mod Folder
![image](https://github.com/Raytwo/ARCropolis/assets/145744391/2bc09e38-e509-4cc1-b8eb-da0e84a7054c)
If inside of the Mod inside the Multi-Mod folder.
![image](https://github.com/Raytwo/ARCropolis/assets/145744391/54f08484-55fc-4f49-9d96-f0fea69eaf00)

(reposted because I thought it was I was posting there on the issues.. I guess wrong category, so posted in the discussions category, that's all)
It's just some idea for me to have you know. Make multiple fighter modpacks instead of a longer job on making multiple fighter addition mods for example.

UPDATE: However, for categorizing, you can even have up to 5 folder layers in the mods folder
let's say: sd:/ultimate/mods/Modpack/Fighter/Slot Addition/Nia
This will be for perfectly clean categorization

**Comment by zrksyd** [comment]
The workspace feature in ARCropolis pretty much accomplishes this already.

**Reply by vistabunny** [reply]
You could fit lots of mods in one folder! instead of being separate in the "sdcard/ultimate/mods" folder

**Reply by vistabunny** [reply]
like Fighter Additions and stuff!

**Reply by zrksyd** [reply]
As good as this would be for organization, it would make ARCropolis pretty inefficient checking for every subfolder. The best workaround would just be to rename your folders so similar mods are next to each other.

**Reply by vistabunny** [reply]
let's hope it would happen which will save much time for modders

**Comment by jozz024** [comment, marked as answer]
Hello. This feature may come out in a future release, but I would not get your hopes up. Also, in the future, please refrain from attempting to contact any developer of the project for any feature requests.

**Reply by vistabunny** [reply]
Thanks for letting me know
